src/redux/data.ts

```diff
@@ -1597,8 +1597,23 @@ export const dataWatchPendingTransactions = (
           }
           const minedAt = Math.floor(Date.now() / 1000);
           txStatusesDidChange = true;
-          // @ts-expect-error `txObj` is not typed as having a `status` field.
-          if (txObj && !isZero(txObj.status)) {
+          let receipt;
```

Comment:
hey QQ: any reason we cannot use `p.getTransactionReceipt(txHash)` directly on line 1588?